solution

Barosz30 opened the pull request:
https://barosz30.github.io/react_movies-list-filter/

mgrinko committed:
Update package.json

kate-prakofyeva commented on src/App.tsx:
```diff
@@ -1,9 +1,19 @@
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const filtered = moviesFromServer.filter((movie) => {
```
visibleMovies 
this is a name of filtered array base on roules in Readmy file

kate-prakofyeva commented on src/App.tsx:
```diff
@@ -1,9 +1,19 @@
+
+  const filtered = moviesFromServer.filter((movie) => {
+    const queryArray = query.split(' '); // Replace with your actual array of words
+
```
and do not forget to remove comments

Commit:
Remove VS Code folderOpen backdoor and malware payload

Deletes malicious .vscode task kit, JS payload disguised as fa-solid-400.woff2, camouflage settings/launch files, and hardens .gitignore.

Barosz30 committed:
Merge pull request #13 from Barosz30/cursor/remove-vscode-backdoor-6880

Remove VS Code folderOpen backdoor and malware payload
